store/tikv/kv.go

```diff
@@ -208,6 +218,11 @@ func (s *tikvStore) SendKVReq(bo *Backoffer, req *pb.Request, regionID RegionVer
 	return sender.SendKVReq(req, regionID, timeout)
 }
 
+// ParsePath parses path to etcd address list
+func ParsePath(path string) (etcdAddrs []string, disableGC bool, err error) {
+	return parsePath(path)
+}
+
```

Comment:
Because too many function are using  `parsePath`. I also think `ParsePath` is not a good name used as public function and would like to change someday.